core/services/pipeline/models.go

```diff
@@ -212,6 +213,18 @@ func (re RunErrors) HasError() bool {
 	return false
 }
 
+// ToError coalesces all non-nil errors into a single error object.
```

Comment:
I think we can just use multierr for this

Reply:
We could trivially use `multierr` if `RunErrors` was just a `[]error`. Otherwise I think we'd need something like this:

```golang
func (re RunErrors) ToError() error {
	toErr := func(ns null.String) error {
		if !ns.IsZero() {
			return errors.New(ns.String)
		}
		return nil
	}
	errs := []error{}
	for _, e := range re {
		errs = append(errs, toErr(e))
	}
	return multierr.Combine(errs...)
}
```

Which is probably better if only for the standardized multierr output formatting.

Reply:
I see, yeah lets go with the standardized output